lib/trello/net/faraday.rb

```diff
@@ -0,0 +1,46 @@
+module Trello
+  module TFaraday
+    class TInternet
+      class << self
+        begin
+          require 'faraday'
+        rescue LoadError
+        end
```

Comment:
We `rescue` this in case the project does not have faraday installed and is using rest-client instead. We do the same in the rest-client version of this class in case the opposite is true.